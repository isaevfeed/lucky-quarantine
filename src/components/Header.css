.Header {
    position: relative;
    font-size: 24px;
    text-align: center;
    padding: 25px 0;
}

.Header>span {
    color: #C2FFBD;
}

.Header:after {
    position: absolute;
    content: "";
    display: block;
    background-color: #464646;
    width: 90%;
    height: 1px;
    bottom: 0px;
    left: 50%;
    transform: translateX(-50%);
}