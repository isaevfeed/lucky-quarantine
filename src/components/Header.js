import './Header.css';
import React from 'react';

const Header = ({}) => (
    <section className="Header">
        Lucky <span>Quarantine</span>
    </section>
);

export default Header;