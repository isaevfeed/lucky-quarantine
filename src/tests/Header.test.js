import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('Header', () => {
    it("render without crashing", () => {
        const div = document.createElement("div");
        ReactDOM.render(<Header />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    
    it("Snapshot: ", () => {
        const header = renderer.create(<Header />).toJSON();
        expect(header).toMatchSnapshot();
    });
});