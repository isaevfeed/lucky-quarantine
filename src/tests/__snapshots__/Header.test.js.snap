// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot:  1`] = `
<section
  className="Header"
>
  Lucky 
  <span>
    Quarantine
  </span>
</section>
`;
